resources/openshift/templates/datavirt64-basic-s2i.json

```diff
@@ -228,7 +275,7 @@
                             "from": {
                                 "kind": "ImageStreamTag",
                                 "namespace": "${IMAGE_STREAM_NAMESPACE}",
-                                "name": "jboss-datagrid71-client-openshift:1.0"
+                                "name": "jboss-datagrid71-client-openshift:1.1"
```

Comment:
@rcernich we won't be releasing those, right?  (so they should keep pointing to 1.0 )

Reply:
Good point.  We should reference the existing client image tags.